scripts/post_install/stub_generation.py

```diff
@@ -42,7 +42,7 @@ def __FindCPPModuleImportsInPythonModules(current_path: Path, cpp_module_names_l
 
                 data = data.replace(" ", "")
                 for i, cpp_module_name in enumerate(cpp_module_names_list):
-                    if "\nfrom{:s}import*".format(cpp_module_name) in data:
+                    if "\nfrom{:s}import*".format(cpp_module_name) in data or "from{:s}import*".format(cpp_module_name) in data:

```

Comment:
you must have known that I was going to suggest regex :laughing:

You could try something like [this](https://regex101.com/r/at6bOp/1) for example:
```suggestion
for match in re.finditer(R"from +(\w+) +import +\*", data):
    cpp_module_name = match.group(1)
    if cpp_module_name in cpp_module_names_list:
```